File: DashboardDemo/DashboardSample/Dashboard/DashboardElement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Telerik.WinControls.UI;
using System.Drawing;

namespace TicketTest
{
    public class DashboardElement: StackLayoutElement
    {
        ColumnElement pending;
        ColumnElement inProgress;
        ColumnElement readyForTest;
        ColumnElement done;
        DashboardDragDropService dragDropService = new DashboardDragDropService();

        public ColumnElement Pending { get { return this.pending; } }
        public ColumnElement InProgress { get { return this.inProgress; } }
        public ColumnElement ReadyForTest { get { return this.readyForTest; } }
        public ColumnElement Done { get { return this.done; } }

        public DashboardDragDropService DragDropService { get { return this.dragDropService; } }

        protected override void InitializeFields()
        {
            base.InitializeFields();

            this.Orientation = System.Windows.Forms.Orientation.Horizontal;
            this.StretchHorizontally = true;
            this.StretchVertically = true;
            this.BackColor = Color.White;
            this.DrawFill = true;
            this.GradientStyle = Telerik.WinControls.GradientStyles.Solid;
            this.DrawBorder = false;
        }

        protected override void CreateChildElements()
        {
            base.CreateChildElements();

            pending = new ColumnElement("Pending");
            this.Children.Add(pending);
            
            inProgress = new ColumnElement("In Progress");
            this.Children.Add(inProgress);
            
            readyForTest = new ColumnElement("Ready for test");
            this.Children.Add(readyForTest);
            
            done = new ColumnElement("Done");
            this.Children.Add(done);
        }
    }
}

File: DashboardDemo/DashboardSampleVB/Dashboard/DashboardElement.vb
Imports Telerik.WinControls.UI

Public Class DashboardElement
    Inherits StackLayoutElement

    Private _pending As ColumnElement
    Private _inProgress As ColumnElement
    Private _readyForTest As ColumnElement
    Private _done As ColumnElement
    Private _dragDropService As DashboardDragDropService = New DashboardDragDropService()

    Public ReadOnly Property Pending As ColumnElement
        Get
            Return Me._pending
        End Get
    End Property

    Public ReadOnly Property InProgress As ColumnElement
        Get
            Return Me._inProgress
        End Get
    End Property

    Public ReadOnly Property ReadyForTest As ColumnElement
        Get
            Return Me._readyForTest
        End Get
    End Property

    Public ReadOnly Property Done As ColumnElement
        Get
            Return Me._done
        End Get
    End Property

    Public ReadOnly Property DragDropService As DashboardDragDropService
        Get
            Return Me._dragDropService
        End Get
    End Property

    Protected Overrides Sub InitializeFields()
        MyBase.InitializeFields()
        Me.Orientation = System.Windows.Forms.Orientation.Horizontal
        Me.StretchHorizontally = True
        Me.StretchVertically = True
        Me.BackColor = Color.White
        Me.DrawFill = True
        Me.GradientStyle = Telerik.WinControls.GradientStyles.Solid
        Me.DrawBorder = False
    End Sub

    Protected Overrides Sub CreateChildElements()
        MyBase.CreateChildElements()
        _pending = New ColumnElement("Pending")
        Me.Children.Add(_pending)
        _inProgress = New ColumnElement("In Progress")
        Me.Children.Add(_inProgress)
        _readyForTest = New ColumnElement("Ready for test")
        Me.Children.Add(_readyForTest)
        _done = New ColumnElement("Done")
        Me.Children.Add(done)
    End Sub
End Class